README.md

```diff
@@ -101,9 +101,10 @@ FUTURE FEATURES
 
 ## 🔭 Future Features <a name="future-features"></a>
 
->
 
-- [ ] **[I would like to add more data in the future]**
+- [ ] Checks if a person has correct name
+- [ ] Capitalizes name
+- [ ] Trims a persons name
```

Comment:
- Kindly add this info to your `Key Feature` section(line 63) and add 1 - 3 features you'll add in the future to this section 👍🏽